packages/layout-grid/Cell.js

```diff
@@ -0,0 +1,46 @@
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Cell = (props) => {
+  const {align, children, className, columns, desktop, order, phone, tablet, ...otherProps} = props;
+
+  const classes = classnames('mdc-layout-grid__cell', className, {
+    [`mdc-layout-grid__cell--align-${align}`]: !!align,
+    [`mdc-layout-grid__cell--order-${order}`]: !!order,
+    [`mdc-layout-grid__cell--span-${columns}`]: !!columns,
+    [`mdc-layout-grid__cell--span-${desktop}-desktop`]: !!desktop,
+    [`mdc-layout-grid__cell--span-${phone}-phone`]: !!phone,
+    [`mdc-layout-grid__cell--span-${tablet}-tablet`]: !!tablet,
+  });
+
+  return (
+    <div className={classes} {...otherProps}>
```

Comment:
div should be configurable. See https://github.com/material-components/material-components-web-react/pull/327/files#diff-a4ae7491ba4f26fa221dde04643b6fa0R7